### PR TITLE
Refactoring and Styling UI Project Using Flexbox

### PR DESCRIPTION
**Pull request by nobioma1**

### CODE REVIEW

File: great-idea/css/index.css [hunk 1]
```diff
@@ -63,4 +63,51 @@ h1, h2, h3, h4, h5 {
...
+	text-align: center;
+	font-size: 60px;
+}
+
+.header-container header a {
+	text-decoration: none;
+	color: gray;
+	padding: 5px 15px;
+	border: 1px solid black;
+}
```
Comment (SorinC6) [inline great-idea/css/index.css:123]: awesome structure of you css file

File: great-idea/css/index.css [hunk 2]
```diff
@@ -162,3 +217,82 @@ footer {
...
+	}
+
+	.more-facts-container {
+		display: flex;
+	}
+
+	.more-facts {
+		padding-right: 20px;
+	}
+}
```
Comment (SorinC6) [inline great-idea/css/index.css:298]: - your media querys look really solid
-adding width with calc() function its awesome

**Comment by SorinC6** [comment]
- Good job on the assignment Noble. Your code is clean. The layout resembles provided design files and Flexbox was used properly.
- Your gitflow is solid !
- Great work on the stretch task : adding those media querys and another page its impressive

**Comment by nobioma1** [comment]
Thank you for the feedback @SorinC6

**Pull request merged**